templates/expo-template-bare-minimum/ios/HelloWorld/AppDelegate.m

```diff
@@ -35,13 +35,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   
   RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"main" initialProperties:nil];
-  id rootViewBackgroundColor = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"RCTRootViewBackgroundColor"];
-  if (rootViewBackgroundColor != nil) {
-    rootView.backgroundColor = [RCTConvert UIColor:rootViewBackgroundColor];
-  } else {
-    rootView.backgroundColor = [UIColor whiteColor];
-  }
-
+  rootView.backgroundColor = [UIColor whiteColor];
```

Comment:
i am getting a little afraid if the view will blink when app running in dark mode. but maybe expo-splash-screen will handle this well. just leaving a note here. this is still looking fine to me.